calendar.html

```diff
@@ -1,79 +1,72 @@
 ﻿<!DOCTYPE html>
 <html>
 
-  <head>
-    <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
-    <title>Calendar of Events</title>
-    <link href="styles/style1.css" media="screen" rel="stylesheet" title="CSS" type="text/css" />
-    <style type="text/css">
-      .auto-style1 {
-        margin-left: 0px;
-      }
-    </style>
-  </head>
+<head>
+  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+  <title>GMVCSC | Calendar of Events</title>
+  <link href="styles/style1.css" media="screen" rel="stylesheet" title="CSS" type="text/css" />
+  <style type="text/css">
+    .auto-style1 {
+      margin-left: 0px;
+    }
+  </style>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>

```

Comment:
Nice 🎉  working on those `jquery` skills here. I like that you're DRYing up the code by moving redundant sections into reusable blocks. that's a good practice.